tests/test_store.py

```diff
@@ -0,0 +1,41 @@
+import os
+import logging
+import pytest
+
+import pandas as pd
+from dotenv import load_dotenv
+
+from schematic.store import BaseStorage
+from schematic.store import SynapseStorage
+
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+
+load_dotenv()
+
+
+@pytest.fixture
+def synapse_store():
+    access_token = os.getenv("SYNAPSE_ACCESS_TOKEN")
+    if access_token:
+        synapse_store = SynapseStorage(access_token=access_token)
+    else:
+        synapse_store = SynapseStorage()
+    return synapse_store
```

Comment:
Maybe this should live in `conftest.py`? Since this fixture will be used by other core modules as well. It's okay to leave it here for now, I can move it in a future PR.

Reply:
I would like to avoid overloading the `conftest.py` files with fixtures that are only used in one test script. 

My philosophy is that each fixture should start in the test script where it's used, and as soon as it would be used in another test script, then it can be moved to `conftest.py`. We can discuss this tomorrow.